elm/TypeAlias.elm

```diff
@@ -44,14 +44,23 @@ capitalize name =
             (String.toUpper (String.fromChar x)) ++ (String.fromList xs)
 
 
+cleanString : String -> String
+cleanString name =
+   let
+       parts = split All (regex "[_.\\- ]") name
```

Comment:
A regex like this can introduce runtime errors - best ot stick at the top level, e.g

```
badCharsRegex = regex "[_.\\- ]"`